syncer/dbconn/db.go

```diff
@@ -160,6 +160,8 @@ func (conn *DBConn) ExecuteSQLWithIgnore(tctx *tcontext.Context, ignoreError fun
 						log.ShortError(err))
 					return false
 				}
+				tctx.L().Warn("execute sql failed by connection error", zap.Int("retry", retryTime),
+					zap.Error(err))
```

Comment:
can we add test for this case?

Reply:
Will add test case for this after we add retry for all other connections so we can safely kill all connection between dm-worker and tidb